app/javascript/src/case_contact.js

```diff
@@ -100,6 +100,13 @@ $(() => { // JQuery's callback for the DOM loading
     caseContactSubmit.prop('disabled', false)
   })
 
+  $('button.toggle-button').on('click', function () {
+    const target = $(this).data('bs-target')
+    const iconId = target.slice(1) + '-icon'
+
+    $(`i#${iconId}`).toggleClass('lni-chevron-up lni-chevron-down')
+  })
+
```

Comment:
Good job implementing the functionality to toggle the chevron up and chevron down CSS class.